### ISSUE TITLE
Compiling error on Ubuntu 14.04 

### ISSUE THREAD

**Issue by jemacom** [Open-Transactions/opentxs#647]
I don't understand why it prompt that "PROTOBUF_INLINE_NOT_IN_HEADERS" is not defined , although I have already install protobuf .
http://pastebin.com/MJ0yTeim

Any help will be appreciated .

**Comment by stevenroose** [comment]
I have the same error on `OTStorage.cpp`, here is the section of the make output:
https://gist.github.com/stevenroose/cd77487297e2279b6973

I also have protobuf installed and opentxs-proto.

**Comment by FellowTraveler** [comment]
See:  http://stackoverflow.com/a/34485004/1115968

**Comment by stevenroose** [comment]
Is that a C++ bug, protobuf bug or opentx bug?

**Comment by FellowTraveler** [comment]
Per the link I posted, it's a protobuf bug, but it doesn't show up on all systems.

I have recently compiled OT for Mac, Linux, and iPhone, and didn't experience the bug you described.

However, it's definitely a real bug – it's been spotted on Linux Mint – you are not the only one who experienced it.

The bug is known, and a fix will be merged soon, but I thought you might want to know the answer in the meantime, so you don't have to wait.
